composer.lock

```diff
@@ -3988,6 +3988,10 @@
                     "recurse": false,
                     "replace": false,
                     "merge-extra": false
+                },
+                "patches_applied": {
```

Comment:
Wait, what? Why is there a patch applied all of a sudden?

Reply:
@pookmish

Reply:
not sure why that shows as "New" changes. these are in the composer.json https://github.com/SU-HSDO/suhumsci/blob/develop/composer.json#L276

Reply:
Did somebody forget to capture it? Keep an eye out for stuff like this.